app/concepts/api/v1/users/registrations/operation/create.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Api::V1::Users::Registrations::Operation
+  class Create < ApplicationOperation
+    step :validate_contract
+    step :create_user
+
```

Comment:
Where step or nested operation with verification email logic?

Reply:
In next PR :)